src/Map.jsx

```diff
@@ -130,6 +131,35 @@ class Map extends Component {
   /**
    * Calculate & Update state of new dimensions
    */
+
+  setNewCity(city) {
+    const cities = [
+      {
+        name: 'San Francisco',
```

Comment:
I see that we're defining the list of cities in two different places. What if we want to add a third city?
Could we define the list of cities in one constant somewhere?